index.bs

```diff
@@ -1307,6 +1313,29 @@ example of the latter, when the user is accessing the [=[RP]=] from a given clie
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
 
+### Attestation Preference enumeration (enum <dfn enum>AttestationPreference</dfn>) ### {#attestation}
+
+[=[RPS]=] may use the {{AttestationPreference}} enum to specify their preference regarding attestation types used 
+during credential generation.
+
+<pre class="idl">
+    enum AttestationPreference {
+        "low-cost",
+        "verifiable" 
+    };
+</pre>
+
+<ul>
+    <li><dfn>low-cost</dfn> - indicates that the [=[RP]=] prefers a low-cost attestation type. For example, if a user agent
+      supports self attestion and Privacy CA, the user agent SHOULD use self attestation, avoiding calls to the Privacy
+      CA, user consent UI, etc. This is the default value.
+    <li><dfn>verifiable</dfn> - indicates that the [=[RP]=] prefers an attestation type that leads to verifiable attestation
+      statements, even if that incurs extra cost such as network roundtrips and/or collection of user consent. For example, 
+      if a user agent supports the self attestation and Privacy CA attestation types, the user agent SHOULD use the Privacy CA
+      attestation type.
```

Comment:
need to describe "direct" attestation enum type as one wehre the client conveys the unadulterated attestation as generated by the authenticator to the RP.

Reply:
Done.